test/testfloat/CMakeLists.txt

```diff
@@ -124,10 +125,10 @@ endif()
 
 function(add_testfloat_selftest NAME ARGS INPUT EXPECTED_ERROR)
     if(INPUT)
-        set(command sh -c "echo ${INPUT} | $<TARGET_FILE:fizzy-testfloat> ${ARGS}")
+        set(command sh -c "echo ${INPUT} | ${EMULATOR} $<TARGET_FILE:fizzy-testfloat> ${ARGS}")
     else()
         separate_arguments(ARGS)
-        set(command $<TARGET_FILE:fizzy-testfloat> ${ARGS})
+        set(command ${CMAKE_CROSSCOMPILING_EMULATOR} $<TARGET_FILE:fizzy-testfloat> ${ARGS})
```

Comment:
Why not `EMULATOR` here?

Reply:
It depends if arguments must be separated with `;` or not.